src/v2i-hub/CARMAStreetsPlugin/src/J3224ToSDSMJsonConverter.h

```diff
@@ -0,0 +1,43 @@
+#include "jsoncpp/json/json.h"
+#include <tmx/j2735_messages/SensorDataSharingMessage.hpp>
+
+namespace CARMAStreetsPlugin
+{
+
+    // Template to use when created shared pointer objects for optional data
```

Comment:
Can we create a TODO comment to move this these functions to a more reusable place since they may be helpful in general for manipulating asn1c generated structs.